client/styles/main.styl

```diff
@@ -61,11 +61,14 @@ nav
     text-indent -9999px
   for $app in $apps
     .{$app}
-      background url('images/'+ $app + '-logo.svg') 40% 50% no-repeat
+      background url('images/' + $app + '-logo.svg') 40% 50% no-repeat
 
   .birt
     background-size 85%
 
+  .performance
```

Comment:
@sturmer-eha it looks like @jgoley was defining logos in a slightly different way by adding them to `$apps` on line three and then appending `-logo.svg` up above.  Jonathan - would you like us to continue using this method or is it ok to define logos using this method?